components/TokenAmountInput/TokenAmountInput.tsx

```diff
@@ -50,6 +50,7 @@ const TokenAmountInput: React.FC<TokenAmountInputProps> = ({
         placeholder="0.0"
         min={0}
         variant={variant}
+        onWheel={(e) => e.currentTarget.blur()}
```

Comment:
would a noop `() => {}` also work here to disable the scroll changes in the number input? or do you specifically want to defocus the input when someone scrolls over it?

Reply:
I tired that but it seems that as long as the element is in focus, the onWheel event would move numbers up/down. Found solution [here](https://stackoverflow.com/questions/63224459/disable-scrolling-on-input-type-number-in-react)